setup.py

```diff
@@ -1,23 +1,35 @@
 from setuptools import find_packages, setup
 
-
-def parse_requirements():
-    with open('requirements.txt', 'r') as f:
-        return f.read().splitlines()
+# The README.md will be used as the content for the PyPi package details page on the Python Package Index.
+with open('README.md', 'r') as readme:
+    long_description = readme.read()
 
 
 setup(
-    name='microengineutils',
-    version='1.0.0',
+    name='microengine-utils',
+    version='1.2.0',
     description='Library for Polyswarm Microengine Utility Package',
+    long_description=long_description,
+    long_description_content_type='text/markdown',
     author='PolySwarm Developers',
     author_email='info@polyswarm.io',
     url='https://github.com/polyswarm/microengine-utils',
     license='MIT',
-    install_requires=parse_requirements(),
+    python_requires='>=3.6,<4',
+    install_requires=[
+        'datadog~=0.35.0',
+        'pytest~=3.9.2',
+    ],
```

Comment:
Looks like `pytest` is not needed for instalation, only for tests.
```suggestion
    install_requires=[
        'datadog~=0.35.0',
    ],
    tests_require=[
        'pytest~=3.9.2',
    ],
```